packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -11,19 +11,27 @@
  * @example
  * import { smoothScroll } from '@carbon/ibmdotcom-utilities';
  *
- * <a href="#anchorlinkname" onClick={smoothScroll(e)}>lorem ipsum</a>
+ * Here e is event param and seletor is param where you want to apply smoothscroll
+ * <a href="#anchorlinkname" onClick={smoothScroll({ e, selector })}>lorem ipsum</a>
  *
  * You can use this for jump to target element by providing event object.
  * it will scroll into view of target by selecting attribute and assigning to id.
  *
- * @param {*} e event object
+ * @param {object} { e: Event, Selector: QuerySelector for the target element }
```

Comment:
Are we gaining any benefits from using an object over two parameters?

Reply:
yeah @photodow  I believe CTA needs an event parameter  and  TOC needs a selector which we need to pass an id.

Reply:
Do we really need two different options, or can we make a decision on a consistent/opinionated way to do it?

Reply:
Sure

Reply:
I think passing an object doesn't seem necessary, seems simpler to be:

`smoothScroll(e, selector)` and not `smoothScroll({e, selector})`

Reply:
@jeffchew first I tried without object `smoothScroll(e, selector) it is not working for TOC . so the reason I wrapped them in an object.